Integrating Product Recs into Page Builder (#508)
magento/merchdocs: src/marketing/page-builder-add-product-recs.md

---
conditions: Default.EE-B2B
title: Add Product Recommendations
---

Use the Product Recommendations content type to add an existing, active recommendation unit to the stage.

## Add an existing recommendation unit

1. Navigate to the Page Builder workspace on the target page.

1. In the Page Builder panel under **Add Content**, drag a **Product Recommendation** placeholder to the stage.

    ![]({% link images/images-ee/page-builder-add-prex-drag.png %}){: .zoom}
    _Select Product Recommendation_

> 1. Hover over the empty product recommendation container. Then, choose **Settings** (![]({% link images/images-ee/icon-pb-settings.png %}){: .Inline}) in the toolbox and do the following:

Reviewer: ```suggestion
1. Hover over the empty product recommendation container, choose **Settings** (![]({% link images/images-ee/icon-pb-settings.png %}) in the toolbox, and do the following:
```

    -  Click <span class="btn">Select</span>.

        ![]({% link images/images-ee/page-builder-add-prex-select.png %}){: .zoom}
        _Select Product Recommendation_

>     -  In the list of active product recommendations, find the row with the recommendation unit that you want to add. In the last column, click <span class="btn">Select</span>.

Reviewer: ```suggestion
    -  In the list of active product recommendations, find the row with the recommendation unit that you want to add and click <span class="btn">Select</span> in the last column.
```

        ![]({% link images/images-ee/page-builder-add-prex-selected.png %}){: .zoom}
        _Selected Product Recommendation_

        The name of the selected product recommendation appears in the **Edit Product Recommendation** form.

        ![]({% link images/images-ee/page-builder-add-prex-name.png %}){: .zoom}
        _Product Recommendation Name_

    -  Complete the remaining settings as needed, using the field descriptions at the end of this page for reference.

    -  When complete, click <span class="btn">Save</span> to return to the Page Builder workspace.

1. In the upper-right corner, click **Save**.

## Edit recommendation unit settings

> 1. Hover over the recommendation unit container. Then, choose **Settings** (![]({% link images/images-ee/icon-pb-settings.png %}){: .Inline}) in the toolbox and do the following:

Reviewer: ```suggestion
1. Hover over the recommendation unit container, choose **Settings** (![]({% link images/images-ee/icon-pb-settings.png %}) in the toolbox, and do the following:
```

    ![]({% link images/images-ee/page-builder-add-prex-toolbox.png %}){: .zoom}
    _Block Toolbox_

>     -  To choose a different recommendation unit, click <span class="btn">Select Product Recommendation</span>. In the list of active recommendation units, <span class="btn">Select</span> the recommendation unit that you want to add. Then, click <span class="btn">Select</span>.

Reviewer: We should not use buttons as verbs in an instruction. You have a `Select` button twice here and I am not sure how this actually works to make a suggestion.

Author: fixed

    -  Update the remaining settings as needed, using the field descriptions at the end of this page for reference.

    -  When complete, click <span class="btn">Save</span> to return to the Page Builder workspace.

1. In the upper-right corner, click <span class="btn">Save</span>.

## Duplicate a recommendation unit

> 1. Hover over the recommendation unit container. Then, choose **Duplicate** (![]({% link images/images-ee/icon-pb-duplicate.png %}){: .Inline}) in the toolbox.

Reviewer: ```suggestion
1. Hover over the recommendation unit container and choose **Duplicate** (![]({% link images/images-ee/icon-pb-duplicate.png %}) in the toolbox.
```

    The duplicate appears just below the original.

> 1. To move the duplicated recommendation unit to a new position, hover over the container. Then, choose **Move** (![]({% link images/images-ee/icon-pb-move.png %}){: .Inline}) in the toolbox.

Reviewer: ```suggestion
1. To move the duplicated recommendation unit to a new position, hover over the container and choose **Move** (![]({% link images/images-ee/icon-pb-move.png %}) in the toolbox.
```

1. Select and drag the recommendation unit until the red guideline appears at the new position.

    The top and bottom borders of each container appear as dashed lines while the recommendation unit is moved.

## Remove a recommendation unit from the stage

> 1. Hover over the recommendation unit container. Then, choose **Remove** (![]({% link images/images-ee/icon-pb-remove.png %}){: .Inline}) in the recommendation unit toolbox.

Reviewer: ```suggestion
1. Hover over the recommendation unit container and choose **Remove** (![]({% link images/images-ee/icon-pb-remove.png %}) in the recommendation unit toolbox.
```
